src/neo/Network/P2P/RemoteNode.cs

```diff
@@ -112,7 +116,21 @@ protected override void OnData(ByteString data)
             msg_buffer = msg_buffer.Concat(data);
 
             for (Message message = TryParseMessage(); message != null; message = TryParseMessage())
+            {
+                if (message_count == 0)
+                    start_message_time = TimeProvider.Current.UtcNow;
+                else if (message_count > 1000)
```

Comment:
1000 txs/per second?

Reply:
We need a reasonable value. It should be bigger.